### PR TITLE
Added set UID size & position

### PR DESCRIPTION
**Pull request by Taiga74164**
thanks @nitrog0d for 🍉

Co-Authored-By: lunatic [56441863+lunaticwhat@users.noreply.github.com](mailto:56441863+lunaticwhat@users.noreply.github.com)

### CODE REVIEW

**Comment by nitrog0d** [comment]
🍉

**Comment by lunaticwhat** [comment]
:wankge:

**Review by lunaticwhat — COMMENTED** [review]
shouldn't be hard coded those values a
on higher resolutions than 1080p like 1440p it will oof (provided video)

solution: 
>get current resolution from ImGui for pos and slap into something like `defaultPos`
>modifying UID Pos X,Y by `defaultPos`+/-float


https://user-images.githubusercontent.com/56441863/177051140-85676813-d7ee-4c37-9640-c3315bfb8d65.mp4

File: cheat-library/src/user/cheat/visuals/ProfileChanger.cpp [hunk 1]
```diff
@@ -34,6 +34,9 @@ namespace cheat::feature
         NF(f_Enabled, "Custom Profile", "Visuals::ProfileChanger", false),
         NF(f_UID, "UID", "Visuals::ProfileChanger", false),
         NF(f_UIDWaterMarkPrefix, "UIDWaterMarkPrefix", "Visuals::ProfileChanger", false),
+        NF(f_UIDsize, "UID Size", "Visuals::ProfileChanger", 14),
+        NF(f_UIDpos_x, "UID Pos X", "Visuals::ProfileChanger", 1190),
```
Comment (lunaticwhat) [inline cheat-library/src/user/cheat/visuals/ProfileChanger.cpp:38]: a

File: cheat-library/src/user/cheat/visuals/ProfileChanger.cpp [hunk 2]
```diff
@@ -34,6 +34,9 @@ namespace cheat::feature
...
+        NF(f_UIDpos_y, "UID Pos Y", "Visuals::ProfileChanger", 0),
```
Comment (lunaticwhat) [inline cheat-library/src/user/cheat/visuals/ProfileChanger.cpp:39]: a

File: cheat-library/src/user/cheat/visuals/ProfileChanger.cpp [hunk 3]
```diff
@@ -62,6 +65,9 @@ namespace cheat::feature
         ConfigWidget(f_Enabled, "Custom Profile.");
         ConfigWidget(f_UID, "Changes the uid visually.");
         ConfigWidget("Append \"UID:\" prefix on the water-mark", f_UIDWaterMarkPrefix);
+        ConfigWidget("UID size", f_UIDsize, 1, 1, 500.0, "Set UID size");
+        ConfigWidget("UID Pos X", f_UIDpos_x, 1.f, 1.f, 1920.f, "Set UID position X");
```
Comment (lunaticwhat) [inline cheat-library/src/user/cheat/visuals/ProfileChanger.cpp:69]: a

File: cheat-library/src/user/cheat/visuals/ProfileChanger.cpp [hunk 4]
```diff
@@ -62,6 +65,9 @@ namespace cheat::feature
...
+        ConfigWidget("UID Pos y", f_UIDpos_y, 1.f, 0, 1080.f, "Set UID position y");
```
Comment (lunaticwhat) [inline cheat-library/src/user/cheat/visuals/ProfileChanger.cpp:70]: a

**Comment by lunaticwhat** [comment]
god damn im highh sorry

**Pull request merged**